parser/regexp.go

```diff
@@ -55,7 +57,7 @@ type _RegExp_parser struct {
 //
 // If the pattern is invalid (not valid even in JavaScript), then this function
 // returns an empty string and a generic error.
-func TransformRegExp(pattern string) (transformed string, err error) {
+func TransformRegExpWithFlags(pattern string, dotAll bool) (transformed string, err error) {
```

Comment:
There is no need for the new function, you can just add the parameter to the existing one,

Reply:
Done!